kafka-connect-s3/src/test/java/io/confluent/connect/s3/integration/S3SinkConnectorNetworkIT.java

```diff
@@ -141,10 +143,9 @@ public void testWithRevokedWritePermissions() throws InterruptedException {
     Thread.sleep(10000);
     // produce more records to kafka
     totalNoOfRecordsProduced += sendRecordsToKafka();
-    int idealObjectCount = totalNoOfRecordsProduced / FLUSH_SIZE;
-    int actualObjectWritten = NUM_RECORDS_PRODUCED / FLUSH_SIZE;
-    assertFalse(assertFileCountInBucket(S3_BUCKET, idealObjectCount).orElse(false ));
-    assertTrue(assertFileCountInBucket(S3_BUCKET, actualObjectWritten).orElse(false));
+    expectedFileCount = totalNoOfRecordsProduced / FLUSH_SIZE;
```

Comment:
can we make this code `totalNoOfRecordsProduced / FLUSH_SIZE` common somehow ?

Reply:
can't make it common as flush size changes for the Pumba test